ENIGMAsystem/SHELL/Widget_Systems/Cocoa/dialogs.m

```diff
@@ -1,4 +1,4 @@
-/** Copyright (C) 2019 Samuel Venable
+/** Copyright (C) 2019 Samuel Venable 
```

Comment:
Get rid of this extra space.

Reply:
the funny thing was, i literally was already did that before I noticed you commented. haha you're quick on the draw there boi

Reply:
@RobertBColton i thought it automatically uses the ci from master? This is new news to me.

Reply:
Yes and no, it compares it to master, but no it will check out the head of the pull request, so unless you pull in master it remains and outdated and will still fail or whatever.

Reply:
Actually, yes and yes. The ci passed. There's nothing left to be discussed and this should be ready to be merged. This is macOS; we don't have an actual test for this platorm? at least one that we don't already ignore completely. This can be merged without doing that crap tbh.